applications/file-split-ftp/pom.xml

```diff
@@ -4,7 +4,6 @@
   <groupId>org.springframework.integration.samples</groupId>
   <artifactId>file-split-ftp</artifactId>
   <version>6.0.0</version>
-  <packaging>pom</packaging>
```

Comment:
The project is based on a Gradle, so if there are some fixes needed they have to be made in the `build.gradle`.
It looks like this packaging is wrong whenever we add Spring Boot Maven plugin (or its parent).

Reply:
I don't see a build.gradle file in the file-split-ftp directory.

Reply:
There is only a single top-level one for the whole project.